### ISSUE TITLE
[Accessibility] VoiceOver does not read last character in code completion suggestions

### ISSUE THREAD

**Issue by guilevi** [microsoft/vscode#120718]
Issue Type: <b>Bug</b>

1. Open a JavaScript file
2. Start typing the word "function" until suggestions pop up. VoiceOver will say "functio, selected" instead of "function, selected"

VS Code version: Code 1.55.0 (c185983a683d14c396952dd432459097bc7f757f, 2021-03-30T16:07:58.402Z)
OS version: Darwin arm64 20.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|undefined|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (0.26GB free)|
|Process Argv|--crash-reporter-id 94a8d63c-71f6-4e0d-a37b-eb3b77b114db|
|Screen Reader|yes|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30276681
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
pythontb:30283811
wslfolderdoc:30282074
vspre833cf:30267465
pythonptprofiler:30281270
vshan820cf:30276953
pythondataviewer:30285071

```

</details>

**Comment by jrieken** [comment]
I can reproduce but not always, e.g at one point it just stopped to reproduce. 

Adding @isidorn this is either an issue with VoiceOver or the list, at least my aria provider is using the same labels as the rendered UI: https://github.com/microsoft/vscode/blob/44955421ae71f1d0c6b0b49414fa6b36aaf6d2cf/src/vs/editor/contrib/suggest/suggestWidget.ts#L226-L236

**Comment by isidorn** [comment]
Yeah I can also reproduce this with VoiceOver and weirdly only with the suggest box. In all other parts of the UI this seems to behave well. I checked multiple lists
Also this works just fine with NVDA.
So this feels like a VoiceOver issue. The best would be if we could repro the issue just with Chrome and VoiceOver so we try to report it to Chrome at least.

@MarcoZehe do you also hit this on macOS?

**Comment by MarcoZehe** [comment]
Yes, I can reproduce that on Mac as well.

**Comment by isidorn** [comment]
Nothing @jrieken should do here, unassigning him.
Leaving up to me to try to repro in Chrome to file an issue

**Comment by isidorn** [comment]
I can no longer reproduce this with latest macOS BigSur and latest VS Code Insiders.
If people still see this please let me know and I will reopen
